client/src/gamedata/gamedata.json

```diff
@@ -446,6 +446,21 @@
       "deployId": "28",
       "instanceGas": 1000000,
       "author": "KStasi"
+    },
+    {
+      "name": "SafeToken",
+      "created": "2020-11-20",
+      "difficulty": "6",
+      "description": "safetoken.md",
+      "completedDescription": "safetoken_complete.md",
+      "levelContract": "SafeTokenFactory.sol",
+      "instanceContract": "SafeToken.sol",
+      "revealCode": true,
+      "deployParams": [],
+      "deployFunds": 0,
+      "deployId": "29",
```

Comment:
This conflicts with https://github.com/OpenZeppelin/ethernaut/pull/546

Reply:
Yes, we can merge this once 546 is merged. I will rebase it with master once 546 is merged.

Reply:
Yes but we need to change deployId to 30